app/services/versioning/file_attachment_revision_updater.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Versioning
+  class FileAttachmentRevisionUpdater < BaseUpdater
```

Comment:
Is the code for this class the same as ImageRevisionUpdater? if so it might be worth having a brain storm with @benthorner on approaches/naming we could use so code and tests aren't duplicated.

Reply:
Yep, I've made a suggestion for this in the PR description.

Reply:
@benthorner and I have had a chat about this.  We felt that renaming the original updater might be confusing and reduce clarity over what type of revision it's for.  We also thought that two similar classes is probably fine for now; if another model comes along that has metadata and a blob and would need a similar class for itself then we could look at extracting out some kind of base metadata and blob revision updater thing and have classes for each of the three models that inherit from the base class.  That's probably overkill right now, but if that need becomes apparent in the future then we can deal with it then.  I'll add this reasoning to the commit message for posterity.